### DISCUSSION TITLE
Previously working animations of USDZ models on iOS broken

### DISCUSSION THREAD

**Discussion by SGECZ** [google/model-viewer#3522] [Q&A]
Take this as an example - it used to work before: https://cldxr.to/nnbkzp2q

In modelviewer the little guy is still walking around - in AR he's stuck - but only on iOS (Safari/Chrome/Firefox)

It used to work before - and we have other models affected - mostly ones with skeletal animation.

Anyone experiencing the same? And/or have a lead to figure this out?

Did anything change on Quicklook that made older USDZ files incompatible?

Thanks you in advance, any help is highly appreciated!

PS: according to https://github.khronos.org/glTF-Validator/ the model is valid

**Comment by elalish** [comment]
Hmm, this looks like a USDZ regression in Quick Look (since you're using a separate `ios-src`, this has nothing to do with your glTF). Perhaps @grorg from Apple can take a look?

**Reply by SGECZ** [reply]
> Hmm, this looks like a USDZ regression in Quick Look (since you're using a separate `ios-src`, this has nothing to do with your glTF). Perhaps @grorg from Apple can take a look?

Right. As an additional information - we use this to convert USDZ files automatically from GLB: https://github.com/google/usd_from_gltf
